src/ucp/core/ucp_context.h

```diff
@@ -263,6 +267,8 @@ typedef struct ucp_context {
 
     char                          name[UCP_ENTITY_NAME_MAX];
 
+    /* Save cached uct configurations */
+    ucs_list_link_t               list;
```

Comment:
maybe rename to "extra_keys_list" or "cached_key_list" ?

Reply:
Done. renamed to "cached_key_list"